### PR TITLE
Add analytics support for GoatCounter

### PR DESCRIPTION
**Pull request by smorkolama**
## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Description
This PR adds support for a privacy-friendly alternative to Google Analytics called GoatCounter (https://www.goatcounter.com). The integration is relatively simple so it should be a low-risk addition. Documentation can be found here: https://www.goatcounter.com/help/start

Additionally I've tested this implementation in my own blog and it works fine. If you have any questions or remarks I will gladly address them of course.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
 - Atom Feeds
 - PWA
 - Google Analytics
+- GoatCounter
```
Comment (cotes2020) [inline README.md:38]:
```suggestion
- Google Analytics / GoatCounter
```

File: _includes/goatcounter.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<!-- GoatCounter -->
+
+<script data-goatcounter="https://{{ site.goatcounter.id }}.goatcounter.com/count"
```
Comment (cotes2020) [inline _includes/goatcounter.html:3]: Prettier format:

```suggestion
<script
  data-goatcounter="https://{{ site.goatcounter.id }}.goatcounter.com/count"
  async
  src="https://gc.zgo.at/count.js"
></script>
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]
It's a nice feature, thanks!

**Comment by smorkolama** [comment]
Thanks for the PR review, I have applied your suggestions!

File: _includes/goatcounter.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<!-- GoatCounter -->
+
+<script
+  data-goatcounter="https://{{ site.goatcounter.id }}.goatcounter.com/count"
+  async
+  src="https://gc.zgo.at/count.js"
+></script>
+        async src="https://gc.zgo.at/count.js"></script>
```
Comment (cotes2020) [inline _includes/goatcounter.html:8]: Looks like this line is redundant.

```suggestion
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Review by cotes2020 — APPROVED** [review]

**Comment by seaque** [comment]
where exactly can we find GoatCounter id?

**Comment by smorkolama** [comment]
> where exactly can we find GoatCounter id?

It's the same value as the subdomain you use to access the goatcounter.com site, in case of https://xyz.goatcounter.com it will be `xyz`

**Pull request merged**